src/Tribe/Views/V2/View.php

```diff
@@ -303,13 +303,13 @@ public static function make_for_rest( \WP_REST_Request $request ) {
 
 		// Determine context based on the request parameters.
 		$do_not_override = [ 'event_display_mode' ];
-		$not_overrideable_params = array_intersect_key( $params, array_combine( $do_not_override, $do_not_override ) );
+		$not_overridable_params = array_intersect_key( $params, array_combine( $do_not_override, $do_not_override ) );
 		$context = tribe_context()
 			->alter(
 				array_merge(
 					$params,
 					tribe_context()->translate_sub_locations( $params, \Tribe__Context::REQUEST_VAR ),
-					$not_overrideable_params
+					$not_overridable_params
```

Comment:
:no_entry_sign: Spaces must be used to indent lines; tabs are not allowed

<sub>hash: 1a1e6802dda702de92725122cdbc3361</sub>